### ISSUE TITLE
[CAIA A11y Research]: Identity Core, SiS - Device Management

### ISSUE THREAD

**Issue by charlye-bluetiger** [department-of-veterans-affairs/va.gov-team#67511]
### Who's on your team?

```bash
Team name: Identity Core
OCTO-DE product owner: Tom Black
Product manager: Mike King
Designer: Charlye Tran
FE engineer: Alex Garcia
Product/team Slack channel: identity-research-public
...
```

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

November 13-17

### Supporting artifacts (if any)

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Product%20Briefs/Sign%20in%20Service.md)
- [User flows](https://app.mural.co/t/oddball4265/m/oddball4265/1694188194725/8b8d5007a442ec71eabcd36941e819eb97dc8968?wid=0-1696604536497)
- [Prototype or mockups](https://app.mural.co/t/oddball4265/m/oddball4265/1694188194725/8b8d5007a442ec71eabcd36941e819eb97dc8968?wid=0-1696604536497)
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Research/2023-09%20Manage_Devices/Research_Plan.md)
- [Research conversation guide](Insert URL here between the parenthesis)
- [Epics/issue that may be helpful](Insert URL here between the parenthesis)
...

### Next steps

- [ ] Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [ ] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [ ] CAIA Specialists are assigned to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
- [ ] CAIA Specialists review your conversation guide and research plan
- [ ] Provide specialists your participant details like skill level, operating system, assistive tech used, browser
- [ ] CAIA specialists will provide you an optional feedback form to let us know how we can support y'all better into the future.
- [ ] CAIA Specialists create a research notes template
- [ ] CAIA Post research, specialists will add accessibility findings to your research folder

**Comment by sara-amanda** [comment]
## CAIA a11y Future Research Sessions Update

- [x] Task lists created for the Identity Core Team and CAIA A11ys in this ticket, with links.
- [x] [Calendar holds have been added](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) for the dates provided: Nov. 13 - 17, 2023.
- [x] Moving to `ready`, as we await official calendar invites for the AT sessions. 

cc: @coforma-terry @SarahKay8 @coforma-jamie

**Comment by coforma-terry** [comment]
Hi @charlye-bluetiger - just checking in to see if you had any updates for your upcoming research sessions planned for Nov 13-17? Thanks !

CC: @sara-amanda

**Comment by coforma-jamie** [comment]
From Clayton on November 6:

> Thanks for checking in! We decided to move forward without AT sessions so we can get a round in before Thanksgiving.
> We’ll plan to regroup after that so we have some more time to set up our prototype for AT sessions.

**Comment by sara-amanda** [comment]
## CAIA Research Update 

### Timing Adjustment
- CAIA was updated that research will not be taking place prior to Thanksgiving 2023, with assistive technology users. 

### Next Steps
- Follow-up needed with the Identity Core Team post-holiday, for updated timing and scheduling of testing. 
- The task lists above have been updated to reflect this change in timing. 
- Ticket title and notes will be updated as well to reflect timing. 
- Moving ticket to CAIA `Backlog`.
- Ref: [DSVA Slack Link 11/6/2023 in #identity-research-public](https://dsva.slack.com/archives/C04S98PE8FK/p1699284989605549?thread_ts=1699281932.336549&cid=C04S98PE8FK)

@coforma-terry @coforma-jamie @charlye-bluetiger

**Comment by coforma-jamie** [comment]
**Update:** Further research rounds are TBD. From Clayton: "We feel the need to regroup and work on next steps with our PM at this point before putting more effort into our present study."

**Comment by coforma-jamie** [comment]
**Update:** We can close out this ticket. From Clayton:
> This work actually just came up this week, We’ve taken some different directions and taken on different bodies of work this year, so we’ve decided to close our team's ticket for this work. So, I think we’re ok to close this one as well.
